src/App.js

```diff
@@ -49,16 +50,17 @@ class App extends React.Component {
         <div className="main-App">
           <div className="form-container">
             <Form getWeather={this.getWeather} />
+            <div className="error"> {this.state.error}</div>
           </div>
 
-          <div class-name="forecast-container">
+          <div class-name="forecast-container0">
```

Comment:
Could you use a better name for this class? I can't see this class defined anywhere in your CSS - if it doesn't exist, don't use it.